packages/pwa-kit-dev/src/ssr/server/build-dev-server.js

```diff
@@ -126,6 +127,44 @@ export const DevServerMixin = {
         })
     },
 
+    /**
+     * @private
+     */
+    _setupExtensions(app, options) {
+        // TODO: support extensions options array syntax i.e. ['extension-a', {}]
+        const extensions = options.mobify?.app?.extensions || []
+        app.__extensions = extensions || []
+
+        extensions.forEach((extension) => {
```

Comment:
We can extend here to support the extensions options array syntax ['extension-name', {}].

```
    extensions.forEach((extensionConfig) => {
        let extensionName, extensionOptions

        if (Array.isArray(extensionConfig)) {
            [extensionName, extensionOptions] = extensionConfig
        } else {
            extensionName = extensionConfig
            extensionOptions = {}
        }
```

Reply:
i'd like to address this in another ticket...